contract/src/platform-goals/zcfTools.js

```diff
@@ -10,20 +11,24 @@ import { atomicRearrange as atomicRearrangeTuples } from '@agoric/zoe/src/contra
  * @param { Array<TransferPartRecord | TransferPart>} transferParts
  *
  * @typedef {{
- *   fromSeat?: ZCFSeat,
- *   toSeat?: ZCFSeat,
- *   fromAmounts?: AmountKeywordRecord,
- *   toAmounts?: AmountKeywordRecord
+ *   from?: ZCFSeat,
+ *   to?: ZCFSeat,
+ *   amounts?: AmountKeywordRecord,
+ *   mappedTo?: KeywordKeywordRecord
```

Comment:
I am surprised by this. When I saw @dtribble 's suggestion that the optional fourth field be named `mappedTo`, it had not occurred to me that it should be a KeywordKeywordRecord for mapping the entries in `amounts`. I was just thinking that it was an alternative to the name `toAmounts`. 

On first blush, both are plausble. As an AmountKeywordRecord, the original meaning of `toAmounts` (whether it is called) is more expressive, but perhaps not usefully so. Independent of naming, what are y'all's thoughts on which is the better API design?

Btw, when I said `mappedTo` grew on me, I only had in mind that it was just a renaming of the original fourth `AmountKeywordRecord` optional argument. But the possibility of having it be a `KeywordKeywordRecord` had not occurred to me. Given that this choice also makes sense, I also agree that the name `mappedTo` is only a name if we choose the `KeywordKeywordRecord` meaning for it.

Reply:
I also don't know which semantics @dtribble had in mind when he suggested `mappedTo`. @dtribble ?

Reply:
Given that the amounts must match, the `KeywordKeywordRecord` has the virtue of not requiring redundant specification, but it also means you can't split up the amounts differently. I think cases where you need to do that are rare, and can be covered in `fromOnly` and `toOnly`.